### PR TITLE
fixing build warnings for OCP 4.12

### PR DESCRIPTION
**Pull request by kalexand-rh**
Nutanix install: https://54738--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned.html#installation-nutanix-config-yaml_installing-nutanix-installer-provisioned
Agent install: https://54738--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer
Operator pod affinity: https://54738--docspreview.netlify.app/openshift-enterprise/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-overriding-operator-pod-affinity_olm-adding-operators-to-a-cluster
API: https://54738--docspreview.netlify.app/openshift-enterprise/latest/rest_api/objects/index.html#io.k8s.apimachinery.pkg.api.resource.Quantity (This matches the live version)

### CODE REVIEW

File: rest_api/objects/index.adoc [hunk 1]
```diff
@@ -5381,19 +5381,14 @@ Quantity is a fixed-point representation of a number. It provides convenient mar
 
 The serialization format is:
 
-``` <quantity>        ::= <signedNumber><suffix>
-
-	(Note that <suffix> may be empty, from the "" case in <decimalSI>.)
-
+<quantity>        ::= <signedNumber><suffix>
```
Comment (kalexand-rh) [inline rest_api/objects/index.adoc:5384]: This is the way this text appears in 4.10 and 4.11.

File: modules/olm-overriding-operator-pod-affinity.adoc [hunk 2]
```diff
@@ -133,7 +133,7 @@ spec:
   sourceNamespace: operator-registries
   config:
     affinity:
-      podAffinity:
+      podAffinity: <1>
```
Comment (kalexand-rh) [inline modules/olm-overriding-operator-pod-affinity.adoc:136]: I double-checked with the author about this callout placement.

File: modules/installation-nutanix-config-yaml.adoc [hunk 3]
```diff
@@ -94,8 +94,8 @@ endif::openshift-origin[]
 If you disable simultaneous multithreading, ensure that your capacity planning accounts for the dramatically decreased machine performance.
 ====
 <4> Optional: Provide additional configuration for the machine pool parameters for the compute and control plane machines.
-ifndef::openshift-origin[]
 <5> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
+ifndef::openshift-origin[]
```
Comment (kalexand-rh) [inline modules/installation-nutanix-config-yaml.adoc:98]: @mjpytlak @mburke5678, any objections here?

Comment (mjpytlak) [reply modules/installation-nutanix-config-yaml.adoc:98]: I encountered the same problem when picking up my Nutanix work this morning after a rebase. Pushing the `ifndef` to line 98 should resolve the error.

Comment (mburke5678) [reply modules/installation-nutanix-config-yaml.adoc:98]: @kalexand-rh My only concern is with the default value. For OKD it is OVNKubernetes; for OCP is it OpenShiftSDN, I do believe.

Comment (mjpytlak) [reply modules/installation-nutanix-config-yaml.adoc:98]: Not sure @mburke5678 looks like the change was made wholesale by https://github.com/openshift/openshift-docs/pull/54319

**Review by skrthomas — COMMENTED** [review]
These updates LGTM

File: modules/installation-nutanix-config-yaml.adoc (hunk 3 above)
Comment (kalexand-rh) [reply modules/installation-nutanix-config-yaml.adoc:98]: @mburke5678, OVNKubernetes appears to be the default as of 4.12, so it should be correct for both OKD and OCP.

**Pull request merged**